openpype/pipeline/frames.py

```diff
@@ -0,0 +1,79 @@
+"""Utility functions.
+
+This module contains utility functions dedicated to frame range operations
+"""
+import clique
+
+from openpype.lib import Logger
+
+log = Logger.get_logger(__name__)
+
+
+def get_frame_start_str(frame_start, frame_end):
+    """Get frame start string.
+
+    Args:
+        frame_start (int): first frame
+        frame_end (int): last frame
+
+    Returns:
+        str: frame start string
+    """
+    padding = len(str(frame_end))
+    return str(frame_start).zfill(padding)
+
+
+def get_asset_frame_range(asset_name=None, asset_id=None, fields=None):
+    """Get the current assets frame range and handles.
+
+    Args:
+        asset_name Optional[str]: Name of asset used for filter.
+        asset_id Optional[Union[str, ObjectId]]: Asset document id.
+            If entered then is used as only filter.
+        fields Optional[Union[List[str], None]]: Limit returned data
+            of asset documents to specific keys.
```

Comment:
Arguments are unused.

I feel like this makes more sense as an abstraction like ```get_asset_frame_range(asset_doc)``` because for moments when you already have the asset document that'd be much faster.

Currently also feels way too much like a function that should be in `context_tools`